src/plone/app/mosaic/transform.py

```diff
@@ -152,10 +152,18 @@ def transformIterable(self, result, encoding):
             ])
 
         # Get contentLayout body class
-        if 'template-layout' in body_classes:
+        if (
+            'template-layout' in body_classes or
```

Comment:
@thet 

        # Get default body classes
        if 'template-' not in body_class and 'site-' not in body_class:
            body_classes.extend([
                n for n in layout.bodyClass(None, self.published).split()
                if n not in body_classes
            ])